README.md

```diff
@@ -1,2 +1,11 @@
-# demo-app
-Repository for the app used in the Rode Demo 
+# Demo Application and CI Pipeline for Rode
+This is the application repository used to demo Rode capabilities.
+- The application used in the demo is a simple Express/Node app serving a `Hello World` response. [server.js](server.js)
+- We build this application Docker image via Kaniko from self-hosted Kubernetes GitHub Action runners.
```

Comment:
Is it worth mentioning that we create a build occurrence after this as well?

Reply:
Yup.